MOBILE_CLIENT/COMMON/_CRITICAL/DEPENDENCY_CONFUSION/context.md

```diff
@@ -0,0 +1,12 @@
+The "Dependency Confusion" vulnerability is a relatively new type of software supply chain attack that was publicized by developer Alex Birsan in 2021. This vulnerability can have significant consequences for the security and integrity of software applications and the companies that develop or rely on them. Here's a breakdown of the risk and impact associated with Dependency Confusion, accompanied by real-world examples:
+
+### Real-World Cases:
+   - A notable instance of Dependency Confusion led to the breach of over 35 major companies' internal systems, including Microsoft, Apple, PayPal, Shopify, Netflix, Yelp, Tesla, and Uber, allowing for remote code execution.
+   - In a publication on February 9, 2021, Alex Birsan demonstrated how he exploited Dependency Confusion to gain access to the internal infrastructure of large companies like Apple and Microsoft.
+   - In 2022, threat actors launched a supply chain attack by injecting a malicious dependency into the PyTorch package.
+
+### Business Impact:
+   - The business impact can be substantial, as seen in the examples above where major companies had their internal systems breached. Such breaches can lead to data leaks, financial loss, and damage to a company's reputation.
+   - A study found that out of the top 1000 organizations scanned, 212 had at least one dependency confusion-related misconfiguration in their codebase, showing that a significant number of organizations may be at risk.
+
+Dependency Confusion is a **serious concern** for software developers and companies relying on open-source packages, underscoring the importance of robust software supply chain security measures.
```

Comment:
Why not make this part of description?

Reply:
The goal is for description to describe the bug from a technical point vue. The context should give non-technical context about the bug, like has been exploited before, is it trending, should I really care.